app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -62,41 +71,55 @@ fun SavedHikesScreen(
       tabList = LIST_TOP_LEVEL_DESTINATIONS,
       selectedItem = Route.SAVED_HIKES,
   ) { paddingValues ->
-    var currentSection by remember { mutableStateOf(SavedHikesSection.Planned) }
+    var currentSection by remember { mutableStateOf(SavedHikesSection.Saved) }
     val loading by savedHikesViewModel.loadingSavedHikes.collectAsState()
     val errorMessageId by savedHikesViewModel.errorMessage.collectAsState()
     val savedHikes by savedHikesViewModel.savedHike.collectAsState()
 
-    val pagerState = rememberPagerState { SavedHikesScreen.values().size }
+    val pagerState = rememberPagerState { SavedHikesSection.values().size }
 
     LaunchedEffect(Unit) { savedHikesViewModel.loadSavedHikes() }
 
     LaunchedEffect(currentSection) { pagerState.animateScrollToPage(currentSection.ordinal) }
 
     LaunchedEffect(pagerState.currentPage, pagerState.isScrollInProgress) {
       if (!pagerState.isScrollInProgress)
-          currentSection = SavedHikesScreen.values()[pagerState.currentPage]
+          currentSection = SavedHikesSection.values()[pagerState.currentPage]
     }
 
-    Column(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
-      // Navigation items between nearby hikes, planned hikes, and saved hikes
-      SavedHikesTabsMenu(
-          selectedIndex = currentSection.ordinal, onSelectedChange = { currentSection = it })
-
-      HorizontalPager(
-          state = pagerState,
-          modifier = Modifier.fillMaxWidth().weight(1f),
-      ) { pageIndex ->
-        Column(modifier = Modifier.weight(1f).testTag(TEST_TAG_SAVED_HIKES_SECTION_CONTAINER)) {
-          SavedHikesScreen.values()[pageIndex].let {
-            when (it) {
-              SavedHikesScreen.Planned -> PlannedHikes(savedHikes)
-              SavedHikesScreen.Saved -> SavedHikes(savedHikes)
-            }
+    Column(
+        modifier =
+            Modifier.fillMaxSize()
+                .padding(paddingValues)
+                .testTag(SavedHikesScreen.TEST_TAG_SAVED_HIKES_SECTION_CONTAINER)) {
+          // Navigation items between nearby hikes, planned hikes, and saved hikes
+          SavedHikesTabsMenu(
+              selectedIndex = currentSection.ordinal, onSelectedChange = { currentSection = it })
+
+          when {
+            loading -> CenteredLoadingAnimation()
+            errorMessageId != null ->
+                CenteredErrorAction(
+                    errorMessage = errorMessageId!!,
+                    actionIcon = Icons.Default.Refresh,
+                    actionContentDescription = R.string.saved_hikes_screen_refresh_button_action,
+                    onAction = { savedHikesViewModel.loadSavedHikes() })
+            else ->
+                HorizontalPager(
+                    state = pagerState,
+                    modifier = Modifier.fillMaxWidth().weight(1f),
```

Comment:
Question: You removed the other weight, but not this one. Is it an omission or required?

Reply:
It's an omission also.